Dockerfile

```diff
@@ -49,13 +49,7 @@ RUN dpkg --add-architecture armel \
 RUN useradd -u1000 -m -d/home/rootless -s/bin/bash rootless
 
 # install bats
-ARG BATS_VERSION
-RUN cd /tmp \
-    && git clone https://github.com/sstephenson/bats.git \
-    && cd bats \
-    && git reset --hard "${BATS_VERSION}" \
-    && ./install.sh /usr/local \
-    && rm -rf /tmp/bats
+RUN npm install -g bats
```

Comment:
Was there a reason for using `npm` for installing, and not from GitHub? the old method (with updated repo) should still work; https://github.com/bats-core/bats-core#installing-bats-from-source

Reply:
Just because it's simpler and someone (hopefully) maintains the package.

Reply:
Not a big fan; not only would we then depend on the `npm` package maintainer to maintain the package (in addition to the new `bats-core` org), it also adds a whole bunch of new dependencies _just_ to install `bats`;

```
docker build -t foo -<<EOF
FROM golang:1.13.10-buster
RUN apt-get update && apt-get install -y --no-install-recommends npm && apt-get clean && rm -rf /var/cache/apt /var/lib/apt/lists/*
EOF

docker history foo
IMAGE               CREATED             CREATED BY                                      SIZE                COMMENT
947f3c9961e5        5 seconds ago       RUN /bin/sh -c apt-get update && apt-get ins…   82.6MB              buildkit.dockerfile.v0
<missing>           5 days ago          /bin/sh -c #(nop) WORKDIR /go                   0B
...
```

That's 83MB to install `npm`, which we only need to install `bats`

Reply:
I recently setup bats for a project, installing from the github repo was very simple.  I would rather us do that instead of using `npm`